<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>611</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="openFile_Button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Open File</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>361</width>
      <height>341</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_averNumMethLines">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>10</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Average number of method lines</string>
    </property>
   </widget>
   <widget class="QPushButton" name="calculate_Button">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>370</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Calculate</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_averNumModuleLines">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>40</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Average number of module lines</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_averNumClassLines">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>70</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Average number of class lines</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_NORM">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>100</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Number Of Remote Methods</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_RFC">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>130</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Response For Class</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_WMC">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>160</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Ð¡lass method complexity</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_NOC">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>190</y>
      <width>211</width>
      <height>201</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string>NOC</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
